src/FrameBuffer.cpp

```diff
@@ -591,6 +595,11 @@ void FrameBufferList::saveBuffer(u32 _address, u16 _format, u16 _size, u16 _widt
 		} else {
 			m_pCurrent->m_resolved = false;
 			glBindFramebuffer(GL_FRAMEBUFFER, m_pCurrent->m_FBO);
+			
+			//Needed for some devices
+			if (_address == gDP.depthImageAddress)
+				glClear(GL_DEPTH_BUFFER_BIT);
```

Comment:
It is wrong functionality. If it fixes issues with some devices, enable it for these devices only.